docs/snapshots/interactive-snapshots.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Interactive Snapshots
+sidebar_label:  Interactive Snapshots
+---
+
+# Interactive Snapshots
```

Comment:
At some point the dashboard UI will have the ability to upload the snaps to cloud workspaces.  I think for now, I'm just going to omit it because it makes a simple concept more complex, and the distinction is covered in the batch snapshots page, where you can do both

Reply:
ok